hedera-node/hedera-app/src/main/java/com/hedera/node/app/grpc/impl/MethodBase.java

```diff
@@ -109,17 +109,17 @@ public abstract class MethodBase implements ServerCalls.UnaryMethod<BufferedData
     @Override
     public void invoke(
             @NonNull final BufferedData requestBuffer, @NonNull final StreamObserver<BufferedData> responseObserver) {
-        try {
-            // Track the number of times this method has been called
-            callsReceivedCounter.increment();
-            callsReceivedSpeedometer.cycle();
-
-            // Fail-fast if the request is too large (Note that the request buffer is sized to allow exactly
-            // 1 more byte than MAX_MESSAGE_SIZE, so we can detect this case).
-            if (requestBuffer.length() > MAX_MESSAGE_SIZE) {
-                throw new RuntimeException("More than " + MAX_MESSAGE_SIZE + " received");
-            }
+        // Track the number of times this method has been called
+        callsReceivedCounter.increment();
+        callsReceivedSpeedometer.cycle();
+
+        // Fail-fast if the request is too large (Note that the request buffer is sized to allow exactly
+        // 1 more byte than MAX_MESSAGE_SIZE, so we can detect this case).
+        if (requestBuffer.length() > MAX_MESSAGE_SIZE) {
+            throw new RuntimeException("More than " + MAX_MESSAGE_SIZE + " received");
```

Comment:
Should we throw this exception or just add a warning ? 
If we throw a RuntimeException here is it be fine ?

Reply:
You are right. The error handling is wrong. I updated the file. Thanks, @Neeharika-Sompalli!